include/Connection.h

```diff
@@ -21,7 +21,7 @@ class Connection {
  public:
     Connection();
     ~Connection();
-    int init(const char* host, uint32_t port, uint32_t weight = 0);
+    int init(const char* host, uint32_t port, const char* alias = NULL);
```

Comment:
The *weight* parameter was defined but is never used neither internally nor externally.


-- 
PAN, Myautsai




On Wed, Aug 12, 2015 at 4:09 AM -0700, "zzl" <notifications@github.com> wrote:












In include/Connection.h:
> @@ -21,7 +21,7 @@ class Connection {
>   public:
>      Connection();
>      ~Connection();
> -    int init(const char* host, uint32_t port, uint32_t weight = 0);
> +    int init(const char* host, uint32_t port, const char* alias = NULL);



Why remove weight ?



—
Reply to this email directly or view it on GitHub.